build.gradle

```diff
@@ -1257,6 +1259,7 @@ project('test:system') {
 
         systemProperty "securityEnabled", System.getProperty("securityEnabled", "false")
         systemProperty "tlsEnabled", System.getProperty("tlsEnabled", "false")
+        systemProperty "tlsCertCNName", System.getProperty("tlsCertCNName", "pravega-pravega-controller")
```

Comment:
pravega-pravega-controller can be defaulted to localhost.

Reply:
Made default as localhost.